wagtail/admin/templates/wagtailadmin/home/locked_pages.html

```diff
@@ -1,34 +1,41 @@
 {% load i18n wagtailadmin_tags %}
 {% load wagtailcore_tags %}
 {% if locked_pages %}
-    {% panel id="locked-pages" heading=_("Your locked pages") %}
-        <table class="listing listing-page">
+    {% panel id="locked-pages" heading=_("Your locked pages") classname="w-panel--dashboard" %}
+        <table class="listing listing--dashboard listing-page">
             <col />
-            <col width="30%"/>
-            <col width="15%"/>
+            <col width="7%"/>
+            <col width="10%"/>
+            <col width="20%"/>
+            <col width="10%"/>
 
             <thead>
                 <tr>
                     <th class="title">{% trans "Title" %}</th>
                     <th aria-hidden="true">{% comment %} added for visual alignment only {% endcomment %}</th>
+                    <th aria-hidden="true">{% comment %} added for visual alignment only {% endcomment %}</th>
+                    <th aria-hidden="true">{% comment %} added for visual alignment only {% endcomment %}</th>
                     <th>{% trans "Locked at" %}</th>
```

Comment:
I think we should review this panel’s columns a bit more. Specifically:

- We could add the page status to fill the available space
- The "Locked at" date matches the date/time column on other panels, not sure if that’s good or not (possibly confusing).